tools/test-builds/browser-agent-wrapper/webpack.config.js

```diff
@@ -68,6 +69,19 @@ const config = [
     output: {
       path: path.resolve(__dirname, '../../../tests/assets/test-builds/browser-agent-wrapper')
     },
+    optimization: {
```

Comment:
Why remove turning off the minimize for test builds?

Reply:
added it back, also added to the other test builds